WordPress/gutenberg
# Adding another test to check on changes to Gallery block.

## What?
1. Adding a new test to visual regression testing.
2. Allows for importing prepared content for easier testing.

## Why?


Visual regression testing often requires some level of content to be created. It is possible, but time-consuming, to create this content using the testing tool. Since Playground makes it trivial to import content, we can take advantage to speed testing and write fewer lines of code.

## How?


The functionality to import content already exists in Playground, and the tests just need to take advantage of it.

## Testing Instructions





1. Open the terminal
1. Run the test suite without a Gutenberg PR: `npm run test:visreg` all tests should pass.
2. Run the test suite with [this PR](https://github.com/WordPress/gutenberg/pull/62762) `PR_NUMBER=62762 npm run test:visreg`
3. All tests fail.

Why do all tests fail? The PR in question is far behind trunk, including visual changes released in WordPress 6.6. However, we can see the specific Gallery changes are picked up in the test.

### Testing Instructions for Keyboard


All of these tests can be run keyboard only

## Screenshots or screencast
